.craft.yml

```diff
@@ -24,3 +24,7 @@ targets:
       - path: /sentry-toolbar/latest/
         metadata:
           cacheControl: 'public, max-age=60'
+  # NPM package target
+  - name: npm
+    id: @sentry/dev-toolbar
```

Comment:
wonder if we use a longform word for `dev` to not confuse it as "in development" e.g. developer-toolbar?